apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -72,101 +70,31 @@ public static void main(String[] args) {
     appMetricsReporter.start(1, TimeUnit.HOURS);
 
     /*
-     * Create the services that will be used to handle each stage in the
-     * extract, transform, and load process.
+     * Create the PipelineManager that will be responsible for running and managing the various
+     * jobs.
      */
-    RifFilesProcessor rifProcessor = new RifFilesProcessor();
-    RifLoader rifLoader = new RifLoader(appMetrics, appConfig.getLoadOptions());
+    PipelineJobRecordStore jobRecordStore = new PipelineJobRecordStore();
+    PipelineManager pipelineManager = new PipelineManager(appMetrics, jobRecordStore);
```

Comment:
Small nit; personally I like to formalize Singleton class by making a private constructor and using a static public getInstance(). For example:

private static PipelineManager myPipelineManager = null;

public static getInstance(MetricRegistry appMetrics, PipelineJobRecordStore jobRecordStore) {
    if (null == myPipelineManager) {
      myPipelineManager = new PipelineManager(appMetrics, jobRecordStore);
   }
  return myPipelineManager;
}

Never have to worry about someone trying to do something they shouldn't!

Reply:
Doesn't that make things more challenging to unit test? I generally have tests that construct things in all sorts of different ways, with various mocked dependencies. Can't do that with a private constructor.

Reply:
Yeah, I definitely ought to `s/must/should` on that one. More of an architecture/design note.